sdk/cosmos/azure-cosmos-benchmark/pom.xml

```diff
@@ -128,7 +128,7 @@ Licensed under the MIT License.
     <dependency>
       <groupId>org.testng</groupId>
       <artifactId>testng</artifactId>
-      <version>6.14.3</version> <!-- {x-version-update;org.testng:testng;external_dependency} -->
+      <version>7.4.0</version> <!-- {x-version-update;org.testng:testng;external_dependency} -->
```

Comment:
testng is deeply used for running all integration tests. If we are upgrading testng version we should run the full test CI and make sure we have a clean Ci result. before merging.